net-misc/networkmanager-fortisslvpn/files/networkmanager-fortisslvpn-1.2.2-location-fix.diff

```diff
@@ -0,0 +1,11 @@
+diff --git a/src/Makefile.am b/src/Makefile.am
```

Comment:
This is a patch that comes from upstream, as noted in the commit message, I'll add a comment to the ebuild too.